ds000117/task-facerecognition_meg.json

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
hmm ... this should actually not be removed. Let me see what needs to be fixed in the validator